frontend/src/App.scss

```diff
@@ -47,14 +42,17 @@ iframe {
   margin: 0;
 }
 
-.streamlit-regular .streamlit-container {
-  margin: 80px auto;
-}
 
-.streamlit-wide .streamlit-container {
-  margin-top: 80px;
-  width: unset;
-  max-width: unset;
+.streamlit-container {
+  &.streamlit-container {
```

Comment:
This becomes `.streamlit-container.streamlit-container`? Why specify the class twice in that way?

Reply:
Or is it a typo and the inner class should say `&.streamlit-regular`?

Reply:
Exactly that!, thanks for the catch